pythran/pythonic/include/utils/numpy_conversion.hpp

```diff
@@ -5,6 +5,14 @@
 
 #include <utility>
 
+#if _MSC_VER
+#define NUMPY_EXPR_TO_NDARRAY0_DECL(fname)                                     \
+  template <class E, class... Types,                                           \
+      typename std::enable_if<                                                 \
+          !types::is_ndarray<E>::value && types::is_array<E>::value,           \
+          E>::type* = nullptr>                                                 \
+  auto fname(E const &expr, Types &&... others);
+#else
```

Comment:
This seems unrelated to the other changes

Reply:
This allow to compile with VS2015/VS2017. This is Windows related :)

Reply:
Otherwise ... 
fatal error C1202: recursive type or function dependency context too complex